app/Http/Controllers/Api/LabelsController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Helpers\Helper;
+use App\Http\Controllers\Controller;
+use App\Http\Transformers\LabelsTransformer;
+use App\Models\Labels\Label;
+use Illuminate\Http\Request;
+use Illuminate\Support\ItemNotFoundException;
+use Auth;
+
+class LabelsController extends Controller
+{
+    /**
+     * Returns JSON listing of all labels.
+     *
+     * @author Grant Le Roux <grant.leroux+snipe-it@gmail.com>
+     * @return JsonResponse
+     */
+    public function index(Request $request)
+    {
+        $this->authorize('view', Label::class);
```

Comment:
I don’t see this permission actually being created or set table anywhere in the code?

Reply:
Is having a permission for this feature something that would be useful to include in this PR?

Reply:
> Adding another JS framework kinda gives me the oogs

Yeah, I get it! I really didn't want to add another one. I tried using Livewire, but it made me sad. I just can't get me head around it to do what I want.

> What’s the difference in page weight

93.8k :(

> is the defer.js part of that?

Sadly it is. Alpine's loading has to be after the DOM. So I needed to mix an "all-defer.js" and load it as <script defer src=...


I'll revisit a Livewire option now that I'm back from holidays.

Reply:
> I don’t see this permission actually being created or settable anywhere in the code?

Oops, nope. It was really just a placeholder in case locking it down was required.

> Is having a permission for this feature something that would be useful to include in this PR? if its not doing anything, it seems like its pretty easy to just remove unless there's a need/desire to have it behind a permission setting

I can remove it for now and reintroduce if needed.